api/src/main/java/org/openmrs/hl7/impl/HL7ServiceImpl.java

```diff
@@ -713,11 +713,8 @@ public HL7InQueue processHL7InQueue(HL7InQueue hl7InQueue) throws HL7Exception {
 		} else {
 			hl7InQueue.setMessageState(HL7Constants.HL7_STATUS_PROCESSING);
 		}
-		
-		if (log.isDebugEnabled()) {
-			log.debug("Processing HL7 inbound queue (id=" + hl7InQueue.getHL7InQueueId() + ",key="
-			        + hl7InQueue.getHL7SourceKey() + ")");
-		}
+	
+			log.debug("Processing HL7 inbound queue (id= {} ,Key= {})", hl7InQueue.getHL7InQueueId(), hl7InQueue.getHL7SourceKey());
```

Comment:
same applies as in comment of api/src/main/java/org/openmrs/hl7/HL7InQueueProcessor.java above